dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat_metrics/test-applications/CDIFaultToleranceMetrics.war/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,25 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+com.ibm.ws.microprofile.faulttolerance_fat.cdi.beans.FallbackBeanWithoutRetry/connectB/Fallback/value=com.ibm.ws.microprofile.faulttolerance_fat.cdi.beans.MyFallbackHandler2
+com.ibm.ws.microprofile.faulttolerance_fat.cdi.beans.FallbackBeanWithoutRetry/connectC/Fallback/fallbackMethod=connectFallback2
```

Comment:
I don't think this file is needed. None of the referenced beans appear to be in the new test project.

Reply:
Fair enough ... will fix under another PR.